apps/gro/fields/address.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  address: {
+    className: ['address'],
+    options: ['']
```

Comment:
I don't think you need `options: ['']`, because at https://github.com/UKHomeOffice/gro/pull/44/files#diff-c7170d1be2e584a31dc0dc9a4b0fdd23R13 you do:
`this.options.fields.address.options = [count].concat(formattedlist);` it's assigning an array to a named new property called `options`.